plugins/reach/lib/api/data/vendorCatalogItem/KalturaVendorLiveCaptionCatalogItem.php

```diff
@@ -5,14 +5,40 @@
  */
 class KalturaVendorLiveCaptionCatalogItem extends KalturaVendorLiveCatalogItem
 {
+	/**
+	 * How long before the live stream start should service activate? (in secs)
+	 * @var int
+	 */
+	public $startTimeBuffer;
```

Comment:
Is there in your opinion a need to add some upper/lower limits to these values?
I don't see any reason why someone can potentially add a value here that is larger than a few hours
So Maybe we should enforce it system side